### PR TITLE
Bump to 5.0, rerender

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please lint.

**Comment by inducer** [comment]
@isuruf, could you explain the changes in 6cf22e3?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,19 +22,13 @@ requirements:
   build:
     - toolchain
     - cmake >=3.4.3
-
-    - system  # [linux and not armv7l]
```
Comment (isuruf) [inline recipe/meta.yaml]: This contains libgfortran and libm, which are not needed for this package. (libm, libstdc++ are assumed to be on the system)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,19 +22,13 @@ requirements:
...
-
-    # "compiling .pyc files" fails without this:
+    # "compiling .pyc files" fails without this
     - python
-
-    - msinttypes  # [win]
-    - dirent      # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: These are not needed in MSVC 2015.

**Comment by inducer** [comment]
@isuruf Thx!

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: That's probably my bad. I think I needed `dirent` before switching from `clang` to `clang-cl`. Should be safe to drop. `msinttypes` should definitely not be needed.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Are any of the LLVM packages still using this? If so, we should definitely clear it out of them as well.

**Comment by jakirkham** [comment]
Should add that we don't build `libcxx` with `libcxxabi` on Windows currently. Not sure if we would like to revisit that. In either case, this would be a good time to think about it.

**Comment by inducer** [comment]
@jakirkham What are the pros/cons? I'm not super in favor, because this is blocking [something else I'm trying to get done](https://github.com/conda-forge/pocl-feedstock/pull/11).

**Comment by jakirkham** [comment]
Sorry for the confusion. Should be more explicit. `libcxxabi` should not be used to target Windows (per upstream). IOW we are already doing the right thing.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -22,19 +22,13 @@ requirements:
   build:
     - toolchain
     - cmake >=3.4.3
-
-    - system  # [linux and not armv7l]
-
-    # "compiling .pyc files" fails without this:
+    # "compiling .pyc files" fails without this
```
Comment (isuruf) [inline recipe/meta.yaml]: This is only true for `clangdev` recipe. Shall I remove python for unix as well?

**Comment by inducer** [comment]
@isuruf Could you do that on a separate PR? Otherwise we'll be back at the end of the queue for Travis.

**Comment by jakirkham** [comment]
That’s probably a good idea for tracking history as well. Though I agree we should do some tidying.

**Comment by isuruf** [comment]
Sure. When travis-ci queue is really long for `conda-forge`, it's okay to merge if travis-ci passes for a fork. https://travis-ci.org/isuruf/libcxx-feedstock/builds/313160748 passes, so I'm okay with merging it now.

**Comment by inducer** [comment]
Alright, thanks!

**Comment by jakirkham** [comment]
Thanks @isuruf and @inducer.

**Pull request merged**